binance_trade_bot/auto_trader.py

```diff
@@ -142,11 +145,21 @@ def _jump_to_best_coin(self, coin: Coin, coin_price: float, all_tickers: AllTick
         # keep only ratios bigger than zero
         ratio_dict = {k: v for k, v in ratio_dict.items() if v > 0}
 
+        can_trade_this_coin = True
+
         # if we have any viable options, pick the one with the biggest ratio
         if ratio_dict:
             best_pair = max(ratio_dict, key=ratio_dict.get)
-            self.logger.info(f"Will be jumping from {coin} to {best_pair.to_coin_id}")
-            self.transaction_through_bridge(best_pair, all_tickers)
+            # Do not allow us to trade with another active coin
+            for active_coin in self.db.get_active_coins():
+                if active_coin.symbol == best_pair.to_coin_id:
+                    can_trade_this_coin = False
+
+            if can_trade_this_coin:
+                self.logger.info(f"Will be jumping from {coin} to {best_pair.to_coin_id}")
+                self.transaction_through_bridge(best_pair, all_tickers)
+            else:
+                self.logger.info(f"--- Skipping trade for {coin}... new coin {best_pair.to_coin_id} is already active")
 
     def bridge_scout(self):
```

Comment:
```suggestion
    def bridge_scout(self, excluded_coins=None):
        if excluded_coins is None:
            excluded_coins = []
```

See my previous comment